serpentTools/tests/test_container.py

```diff
@@ -0,0 +1,86 @@
+"""Test the container object. """
+
+import unittest
+from serpentTools.objects import containers
+from serpentTools.parsers import DepletionReader
+
+
+class containerTester(unittest.TestCase):
```

Comment:
Change the name of this class to reflect the container being tested. We have/will have many containers down the road

Reply:
Done. Changed in "HomogenizedUniverseTester"